health-services/project/src/main/java/org/egov/project/Constants.java

```diff
@@ -56,4 +56,18 @@ public interface Constants {
 
     String TASK_QUANTITY = "taskQuantity";
 
+    String HOUSEHOLD_ID = "HouseholdId";
+
+    String SET_USER_ACTION = "setUserActions";
```

Comment:
Do we need this? are we still using reflections for these ?

Reply:
yes, it is required as reflection is still used as we don't have a generic search object

Reply:
*request object